packages/loader/src/resource-deserialize/resources/scene/EditorTextureLoader.ts

```
@@ -1,19 +1,43 @@
-import { AssetPromise, Loader, LoadItem, resourceLoader, ResourceManager, Texture2D } from "@galacean/engine-core";
+import { AssetPromise, ContentRestorer, Loader, LoadItem, resourceLoader, ResourceManager, Texture2D } from "@galacean/engine-core";
```

ESLint

[error] Replace `·AssetPromise,·ContentRestorer,·Loader,·LoadItem,·resourceLoader,·ResourceManager,·Texture2D·` with `⏎··AssetPromise,⏎··ContentRestorer,⏎··Loader,⏎··LoadItem,⏎··resourceLoader,⏎··ResourceManager,⏎··Texture2D⏎`

(prettier/prettier)